test/llvm.assume.ll

```diff
@@ -0,0 +1,110 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
This file should be in `transcoding` subdirectory, because it performs translation in both ways